config/samples/flows_v1beta1_flowcollector.yaml

```diff
@@ -137,3 +137,10 @@ spec:
     #   kafka:
     #     address: "kafka-cluster-kafka-bootstrap.netobserv"
     #     topic: netobserv-flows-export
+    # or
+    # - type: IPFIX
+    #   ipfix:
+    #     targetHost: "ipfix-collector.ipfix.svc.cluster.local"
+    #     targetPort: 4739
+    #     transport: tcp
+    #     enterpriseId: integer number (optional) 
```

Comment:
:+1: perfect !

Reply:
Actually not perfect. Forgot to change the tcp part. It can be udp and also optional.